utils.js

```diff
@@ -60,7 +60,9 @@ function ecsMeta(req, res) {
     }
 
     if (req.headers['content-length']) {
-      meta.http.request.body.bytes = req.headers['content-length'];
+      meta.http.request.body = {
```

Comment:
Um... but that will completely override the. body, no?

Reply:
mm yeah, but http.request.body doesn't exists.